modules/images/webp-uploads/load.php

```diff
@@ -31,6 +31,11 @@ function webp_uploads_filter_image_editor_output_format( $output_format, $filena
 		return $output_format;
 	}
 
+	// Skip conversion when creating the `-scaled` image (for large image uploads).
+	if ( 1 === preg_match( '/.*-scaled\..{3}.?$/', $filename ) ) {
```

Comment:
The `1 === ` is maybe a bit overly strict, we can probably remove it. I'm not sure, but I'm thinking maybe some obscure PHP version returns an integer here, another a boolean - in any case, for simple boolean(-ish) checks I don't think we need to have strict checks, similar to how it's done in other such places in WP core.

Reply:
👍🏼  updated